tools/rdm/TestDefinitions.py

```diff
@@ -777,6 +788,24 @@ def VerifyResult(self, response, fields):
         self.AddAdvisory('%s supported but %s is not' %
                          (pid_names[0], ','.join(unsupported_pids)))
 
+    for pid_names in self.PID_REVERSE_DEPENDENCIES:
```

Comment:
then you can write this as

for required_pid, other_pids in self.PID_REVERSE_DEPENDECIES: